src/compas_blender/utilities/drawing.py

```diff
@@ -80,16 +80,16 @@ def _set_object_color(obj, rgb, alpha=1.0):
 
 def draw_texts(texts: List[Dict],
```

Comment:
i guess we should start replacing thee by `TypedDict` (from `typing_extensions` for < 3.8) or a `NamedTuple`

Reply:
you mean by rewriting `color_to_colordict`?